src/php/composer.json

```diff
@@ -1,20 +1,23 @@
 {
-  "name": "grpc/grpc-dev",
-  "description": "gRPC library for PHP - for Developement use only",
+  "name": "grpc/grpc",
+  "type": "library",
+  "description": "gRPC library for PHP",
+  "keywords": ["rpc"],
+  "homepage": "https://grpc.io",
   "license": "Apache-2.0",
-  "version": "1.7.0",
```

Comment:
We need this. https://github.com/grpc/grpc/blob/master/src/php/lib/Grpc/BaseStub.php#L68

I prefer we keep this file as is. This is basically useful only for internal development (as described by the `description` field above). We can update the `google/protobuf` to `^3.4.0`.

Reply:
Ahh woops, sorry I'll add that back.

We DO want to update the name and description, however, and add the suggestion. I'll fix the rest!